index.html

```diff
@@ -2859,8 +2859,8 @@ <h3>Production</h3>
         <p>
 The <a>DID document</a> MUST be serialized according to the
 <a href="#production">production rules for JSON</a>, with one additional
-requirement: The <a>DID document</a> MUST include the <code>@context</code>
-entry.
+requirement: The serialized <a>DID document</a> MUST include the
```

Comment:
Can we debate these changes in https://github.com/w3c/did-core/pull/626 and address the other production rules by themselves.